packages/devtools_app/lib/src/screens/dtd/events.dart

```diff
@@ -0,0 +1,223 @@
+// Copyright 2025 The Flutter Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
+
+import 'dart:async';
+
+import 'package:devtools_app_shared/ui.dart';
+import 'package:devtools_app_shared/utils.dart';
+import 'package:dtd/dtd.dart' show DartToolingDaemon, DTDEvent;
+import 'package:flutter/material.dart';
+
+import 'dtd_tools_model.dart';
+import 'shared.dart';
+
+class EventsController extends FeatureController {
+  late DartToolingDaemon dtd;
+
+  final _events = ListValueNotifier<DTDEvent>([]);
+
+  final _selectedEvent = ValueNotifier<DTDEvent?>(null);
+
+  final scrollController = ScrollController();
+
+  @override
+  void init() {
+    super.init();
+    for (final stream in knownDtdStreams) {
+      autoDisposeStreamSubscription(
+        dtd.onEvent(stream).listen((event) {
+          _events.add(event);
+          // Schedule a scroll to the bottom after the frame is built
+          unawaited(
+            Future.microtask(() {
```

Comment:
Should this be using [`addPostFrameCallback`](https://api.flutter.dev/flutter/scheduler/SchedulerBinding/addPostFrameCallback.html) instead?